src/Illuminate/Database/DatabaseManager.php

```diff
@@ -280,7 +280,12 @@ public function supportedDrivers()
      */
     public function availableDrivers()
     {
-        return array_intersect($this->supportedDrivers(), PDO::getAvailableDrivers());
+        return array_unique(
+            array_intersect(
+                $this->supportedDrivers(),
+                preg_replace('/^dblib$/', 'sqlsrv', PDO::getAvailableDrivers())
```

Comment:
I considered str_replace but preg_replace made more sense (IMO) in case another driver where dblib is a substring of the name ever shows up in PDO. Can you think of a better way?